client/Windows/wf_floatbar.c

```diff
@@ -125,7 +125,7 @@ static BOOL floatbar_animation(wfFloatBar* const floatbar, const BOOL show)
 
 	floatbar->animating = timer;
 
-	if (SetTimer(floatbar->hwnd, timer, USER_TIMER_MINIMUM, NULL) == NULL)
+	if (SetTimer(floatbar->hwnd, timer, USER_TIMER_MINIMUM, NULL) == 0)
 	{
```

Comment:
This change is strange as `SetTimer` returns a pointer

Reply:
`SetTimer()` returns `UINT_PTR` type which is `unsigned int` (32 bit) or `unsigned __int64` (64bit) according to `shared/basestd.h` in Microsoft Windows SDK.